### PR TITLE
Update install-node.md

### PR DESCRIPTION
**Pull request by adrpan**
update libssl to actual link and some outputs due very outdated...

### CODE REVIEW

File: source/docs/casper/operators/setup/install-node.md [hunk 1]
```diff
@@ -31,10 +31,11 @@ casper-client : Depends: libssl1.1 (>= 1.1.0) but it is not installable
 ```
 
 This message is due to the default `openssl` moving to 3.* with Ubuntu 22.04. You need to install OpenSSL 1.* for prior versions of Ubuntu to use the Casper binaries with the following command:
+https://packages.ubuntu.com/focal/amd64/libssl1.1/download
```
Comment (ipopescu) [inline source/docs/casper/operators/setup/install-node.md:34]: Please remove the standalone link; it's a bit confusing.
```suggestion
This message is due to the default `openssl` moving to 3.* with Ubuntu 22.04. You need to install [OpenSSL 1.*](https://packages.ubuntu.com/focal/amd64/libssl1.1/download) for prior versions of Ubuntu to use the Casper binaries with the following command:
```

File: source/docs/casper/operators/setup/install-node.md [hunk 2]
```diff
@@ -31,10 +31,11 @@ casper-client : Depends: libssl1.1 (>= 1.1.0) but it is not installable
...
 
 ```
-curl -f -JLO http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb
-sudo apt install ./libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
+wget http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.21_amd64.deb
+sudo dpkg -i ./libssl1.1_1.1.1f-1ubuntu2.21_amd64.deb
```
Comment (ipopescu) [inline source/docs/casper/operators/setup/install-node.md:38]: I suggest keeping the previous commands, unless you can explain why we should change them.
```suggestion
curl -f -JLO http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.21_amd64.deb
sudo apt install ./libssl1.1_1.1.1f-1ubuntu2.21_amd64.deb
```

**Review by ipopescu — CHANGES_REQUESTED** [review]
Great updates. I've requested some changes though.

**Comment by ACStoneCL** [comment]
Your build is failing due to an external link problem from our end - if you update your branch, it should solve the problem.

File: source/docs/casper/operators/setup/install-node.md (hunk 2 above)
Comment (adrpan) [reply source/docs/casper/operators/setup/install-node.md:38]: sure it can be. I prefer install single packages via dpkg instead apt

**Comment by ipopescu** [comment]
@adrpan could you please update this PR so we can take it forward?

**Comment by ipopescu** [comment]
Closing this PR as I don't feel comfortable merging it. The link provided doesn't exist. If the changes need to be made, please retest them and submit a new PR.

**Pull request closed without merging**